gui/locales/README.md

```diff
@@ -20,9 +20,6 @@ procedure, described in the section below.
 Run `npm run update-translations` to extract the new translations from the source
 code and update all of the existing catalogues.
 
-The new translations are automatically added to empty sub-folders using the POT template at
```

Comment:
Basically this statement still applies when you wish to produce automatically translated `relay-locations.po` before uploading it to Crowdin. So like adding an empty folder with named after the new locale identifier and running the `extract-geo-data.py` should attempt to auto-translate the relay-locations into the target language.